src/types/User.tsx

```diff
@@ -0,0 +1,80 @@
+export interface User {
+  id: number,
+  name: string;
+  username: string;
+  email: string;
+}
+
+// export const App = () => {
+//   const [goods, setGoods] = useState<Good[]>(goodsWithColor);
+//   const [newGoodName, setNewGoodName] = useState('');
+//   const [selectedColorId, setSelectedColorId] = useState(0);
+//   const [hasNameError, setHasNameError] = useState(false);
+//   const [hasColorError, setHasColorError] = useState(false);
+
+//   const addNewGood = (name: string, colorId: number) => {
```

Comment:
Consider renaming `addNewGood` to `addNewProduct` to match the naming convention of the rest of the code. 🤔